cabal.project.stage2

```diff
@@ -45,6 +45,9 @@ packages:
   utils/hp2ps
   utils/runghc
   utils/unlit
+  utils/haddock
+  utils/haddock/haddock-api
+  utils/haddock/haddock-library
```

Comment:
Again we only need haddock for the test-suite, right?

Reply:
Haddock should be part of the bindist.